lib/mutant/mutator/node/send.rb

```diff
@@ -77,6 +77,8 @@ def meta
         private :arguments
 
         def normal_dispatch
+          return if meta.sorbet_receiver?
```

Comment:
With this we only leave the singleton mutation as far as I could tell.

Reply:
I think the singletons should also be avoided in this case.

Reply:
Oh right, they would die always (for `T.must` at least) anyway.